language-design.md

```diff
@@ -102,6 +201,18 @@ you do so? If not, why not?
 
 **Response**
 
+We think that this shows that there is a balance to be struck between natural
+language design and regular scripting language syntax.  It is also important
+to know the audience of the language.  If they are experienced programmers it
+is probably easier for them if your programming language looks more like what
+they are used to and has the less natural syntax of Java or C, but if they are
```

Comment:
This seems to make the assumption that experienced programmers are used to this style of syntax (and seems to, at least slightly, fall into the Y&R hole). There are certainly some experienced programmers who mostly use languages like Perl, Ruby, Python, etc, and for whom Java/C are much harder to use.